src/objects/core/zcl_abapgit_filename_logic.clas.abap

```diff
@@ -88,6 +91,8 @@ CLASS zcl_abapgit_filename_logic IMPLEMENTATION.
   METHOD object_to_file.
 
     DATA lv_obj_name TYPE string.
+    DATA lv_nb_of_slash TYPE string.
+    DATA lo_aff_registry TYPE REF TO zif_abapgit_aff_registry.
```

Comment:
let's make this a class variable `go_aff_registry` and instantiate it in the class constructor.